web/src/types/TestOutput.types.ts

```diff
@@ -0,0 +1,19 @@
+import {Model, TTestSchemas} from './Common.types';
+
+export type TRawTestOutput = TTestSchemas['TestOutput'];
+
+export type TTestOutput = {
```

Comment:
Do we still need this? I think it would be better to compose the type from the open API spec and overwrite the fields we need, that we can avoid changing it "that" often

Reply:
Yeah, hmm that was my first approach. I can change that, I don't have a strong opinion about it. However not a big fan of using the `Model<>` type. I think it makes "not easy" to understand your types when you are developing.

Reply:
Not a blocker, let's discuss the `Model<>` approach to see what we can do for the future